scripts/generate-open-api-spec

```diff
@@ -161,7 +159,7 @@ function runOpenAPIGenerate() {
     configFile="$(basename -- ${configLocation})"
   fi
 
-  genSpecCmd="docker run --rm  --user $(id -u):$(id -g) -v ${hostSharedDir}:${OPEN_API_MOUNT}"
+  genSpecCmd="docker run --rm  --user 0:0 -v ${hostSharedDir}:${OPEN_API_MOUNT}:z"
```

Comment:
I was able to run it, but hit three things:

- A pile of errors that I think are expected (below) plus a pile of warnings.

```
##########################################################################################
#
#  Starting openapi code generation with command: 
         docker run --rm --user 0:0 -v /home/swcur/repos/aries-cloudagent-python/scripts/../open-api/.build:/local:z docker.io/openapitools/openapi-generator-cli:v6.6.0 generate -g openapi --input-spec /local/acapy-raw.json --output /local --config /local/openAPIJSON.config
#
##########################################################################################

[main] WARN  o.o.c.config.CodegenConfigurator - There were issues with the specification, but validation has been explicitly disabled.
Errors: 
        -attribute paths.'/issue-credential/records'(get).[connection_id].example is unexpected
        -attribute paths.'/credential/revoked/{credential_id}'(get).[from].example is unexpected
        -attribute paths.'/discover-features-2.0/queries'(get).[query_goal_code].example is unexpected
```

- Unable to delete some files at the end of the process because of permissions issue — presumably because the folder did not exist until the docker process created them with root ownership.  I’m guessing we can ignore that for now, but better create the folder ahead with “inherit” settings?

```
rm: cannot remove '/home/swcur/repos/aries-cloudagent-python/scripts/../open-api/.build/.swagger-codegen/VERSION': Permission denied
rm: cannot remove '/home/swcur/repos/aries-cloudagent-python/scripts/../open-api/.build/.openapi-generator/VERSION': Permission denied
rm: cannot remove '/home/swcur/repos/aries-cloudagent-python/scripts/../open-api/.build/.openapi-generator/FILES': Permission denied
```

3. After I generated, the “connection state” enumeration had the same values, but in different an different order. I assume we can ignore that.

Reply:
Thanks for the test! I'll look into the permissions thing. The fix you suggest seems like the right way to go.

Reply:
Interesting... Thanks for the data point!